pallets/avn/src/lib.rs

```diff
@@ -296,13 +361,9 @@ impl<T: Config> Pallet<T> {
         });
     }
 
-    pub fn calculate_two_third_quorum() -> u32 {
-        let len = Self::active_validators().len() as u32;
-        if len < 3 {
-            return len
-        } else {
-            return (2 * len / 3) + 1
-        }
+    pub fn calculate_one_third_quorum() -> u32 {
```

Comment:
we seem to have this function defined twice. Once in Primitives and another here.

Reply:
Removed from the avn pallet, everything references the primitives implementation now